File: 0708/introc/sumatorio.py
# -*- coding: utf-8 -*-
from math import *

# Pedir l´ımites inferior y superior.

a = int(raw_input('Límite inferior: '))

while a < 0:
    print 'No puede ser negativo'
    a = int(raw_input('Límite inferior: '))
    
b = int(raw_input('Límite superior:'))
while b < a:
    print 'No puede ser menor que %d' % a
    b = int(raw_input('Límite superior:'))

# Calcular el sumatorio de la ra´ız cuadrada de i para i entre a y b.
s = 0.0

for i in range(a, b+1):
    s += sqrt(i)

# Mostrar el resultado.
print 'Sumatorio de raíces',
print 'de %d a %d: %.2f' % (a, b, s)

File: trunk/0708/introc/sumatorio.c
#include <stdio.h>
#include <math.h>

// siempre función principal: main 
/* devuelve entero, void: indeterminado */
int main(void)
{
   /* obligatorio marcar inicio y final de bloque con {} */
   
   int a, b, i;  /* obligatorio declarar variables: 
                    tipo y nombre*/ 
                    /* sentencias acaban con ;  */
   float s;
   
   /* límites */
   printf("Límite inferior: ");
   scanf("%d", &a);  /* lee de la pantalla.
                     Indicar: tipo de dato y dirección de var */
   while (a < 0){
         printf("No puede ser negativo\n");
         printf("Límite inferior: ");
         scanf("%d", &a);
   }
   
   printf("Límite superior: ");
   scanf("%d", &b); 
   
   while (b < a){
         printf("No puede ser menor que %d", a);
         printf("Límite superior: ");
         scanf("%d", &b);
   }
   
   /* cálculo del sumatorio */
   s = 0.0;
   for (i = a; i<=b; i++){
       s += sqrt(i);
       }
   printf("Sumatorio de raíces ");
   printf("de %d a %d: %.2f \n", a, b, s);
   
   system("pause");
   return 0;
}
